docs/drill4net/drill4net-apps.mdx

```diff
@@ -37,7 +37,6 @@ Table below describes each setting.
 | **communicatorUrl**                                           | Communication URL of Drill4J Admin back-end.                                                                                                                                                       |
 | **coverageCollectionSettings:  schedulerDelayInterval**       | Interval in seconds indicating how often code coverage data is sent to the Drill4J Admin. The default value is 10 seconds.                                                                         |
 | **coverageCollectionSettings:  collectInitProcessesCoverage** | Specifies whether to collect coverage for initial processes (Background jobs, Initialization, Cron). The default value is false.                                                                   |
-| **supportedTechnologies**                                     | An array of supported technologies in application under test. Currently, the only available value is **HttpClient**.                                                                               |
```

Comment:
Currently, for other supported technologies for transport such as Kafka, Azure Service Bus, and Rabbit MQ the user calls the patch manually in the Startup class. So, the supportedTechnologies parameter in Scanner is not used. For HttpClient as transport, the Patch is called automatically if the user adds value for the serviceGroupId parameter

Reply:
> For other supported technologies for transport such as Kafka, Azure Service Bus, and Rabbit MQ the user calls the patch manually in the Startup class. 

@AndriiTomashevskyi  is it specified _anywhere_ in the docs?
If not, make sure to add that information with the code example

Reply:
Yes, serviceGroupId is described in the configuration table;
https://drill4j.github.io/docs/drill4net/drill4net-apps#3-configuration
How to set up each supported technology is described on app-under-test-setup page:
https://drill4j.github.io/docs/drill4net/app-under-test-setup